virttest/libvirtd_decorator.py

```diff
@@ -0,0 +1,129 @@
+"""
+A decorator utility functions to apply libvirtd functions.
+
+Copyright: Red Hat Inc. 2020
+"""
+import logging
+import os
+import re
+from virttest import data_dir
+
+from avocado.utils import process
+from avocado.utils import path
+from avocado.utils import astring
+
+
+try:
+    path.find_command("libvirtd")
+    LIBVIRTD = "libvirtd"
+except path.CmdNotFoundError:
+    LIBVIRTD = "virtqemud"
+
+
+def get_libvirtd_split_enable_bit():
+    base_cfg_path = os.path.join(data_dir.get_shared_dir(), 'cfg', 'base.cfg')
+    if os.path.isfile(base_cfg_path):
+        with open(base_cfg_path, 'r') as base_file:
+            for line in base_file:
+                if 'enable_split_libvirtd_feature' in line and 'yes' in line:
```

Comment:
This will evaluate to `True` even if the line is commented out, and it shouldn't.  Overall, I really suggest a different approach for this conditional behavior.

Reply:
1)try to use cartesian_config to extract value from /home/chunfu/workdir/p3-v3/code/avocado-vt/shared/cfg/base.cfg, and finding that cartesian_parser.get_dicts() is not relaible ,and return None after execute second time. 

>>> import os
>>> from virttest import data_dir
>>> from virttest import cartesian_config
>>> 
>>> base_cfg_path = os.path.join(data_dir.get_shared_dir(), 'cfg', 'base.cfg')
>>> print(base_cfg_path)
/home/chunfu/workdir/p3-v3/code/avocado-vt/shared/cfg/base.cfg
>>> cartesian_parser = cartesian_config.Parser()
>>> cartesian_parser.parse_file(base_cfg_path)
>>> 
>>> 
>>> dicts = cartesian_parser.get_dicts()
>>> for params in dicts:
...     split_feature = params.get('enable_split_libvirtd_feature')
...     print ("first time is :%s" % split_feature)
... 
first time is :no
>>> for params in dicts:
...     split_feature = params.get('enable_split_libvirtd_feature')
...     print ("second time is :%s " % split_feature)
... 
>>> 
2)

Reply:
Anyway, although try above two solutions but it sound like they both work as expected. So just keep current implementation and add additional checking that there is no "#" in the line.